test/scripts/health_check.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#example usage: test/scripts/health_check.sh --network=main --height=52000 --version=2.1.0 18.136.37.63
+set -eo pipefail
+
+main_network_id=ae_mainnet
+uat_network_id=ae_uat
+main_genesis="kh_pbtwgLrNu23k9PA6XCZnUbtsvEFeQGgavY4FS2do3QP8kcp2z"
+uat_genesis="kh_wUCideEB8aDtUaiHCtKcfywU6oHZW6gnyci8Mw6S1RSTCnCRu"
```

Comment:
We can hardcode main and uat minimum hight as well

Reply:
what number ?